docs/diagrams/ModelClassDiagram.puml

```diff
@@ -47,6 +53,12 @@ Company *--> CareerPageUrl
 Company *--> Address
 Company *--> "*" Tag
 
+Tag <|-- SalaryTag
```

Comment:
Good modelling of the different tags!

Reply:
This certainly makes the class diagram clearer and more accurate!